sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1990,6 +1995,30 @@
           "customOptions": {},
           "useExactSearch": false,
           "id": "eiulpp"
+        },
+        {
+          "key": "institutionEnteredDetailsStudyperiodwarning",
```

Comment:
Please adjust the case `institutionEnteredDetailsStudyPeriodWarning`.

Reply:
It is the API key of HTML component(Banner). Do we need to change that?